src/components/Exchange/ConfirmationBox.js

```diff
@@ -378,16 +318,18 @@ export default function ConfirmationBox(props) {
       4,
       true
     );
+    const longOrShortText = select(existingOrder.isLong, { true: "Long", false: "Short" });
```

Comment:
wondering why we need `select` in this case? is it the same?

```
const longOrShortText = existingOrder.isLong ? t`Long` : t`Short`?
```

just looks a bit simpler imo

Reply:
like here https://github.com/gmx-io/gmx-interface/pull/321/files#diff-d8826fb03229e798252f7c2101f6fbae00a01996892c9f7337d8eb147465b1fbR332

Reply:
<img width="481" alt="image" src="https://user-images.githubusercontent.com/95585424/197542212-9b726747-d431-426a-8c69-2d5dd32d3b9d.png">

Reply:
Makes sense👍 I was using `select` when we can have more than 2 options. Here it will never have more than 2 options.